### PR TITLE
Fix golint failures in pkg/registry/admissionregistration/...

### PR DESCRIPTION
**Pull request by xichengliudui**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:
Fix golint failures in pkg/registry/admissionregistration/...

**Which issue(s) this PR fixes**:
Ref https://github.com/kubernetes/kubernetes/issues/68026

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/quota/v1/evaluator/core/doc.go [hunk 1]
```diff
@@ -15,4 +15,4 @@ limitations under the License.
 */
 
 // core contains modules that interface with the core api group
-package core // import "k8s.io/kubernetes/pkg/quota/v1/evaluator/core"
+Package core // import "k8s.io/kubernetes/pkg/quota/v1/evaluator/core"
```
Comment (xichengliudui) [inline pkg/quota/v1/evaluator/core/doc.go]: why change: using verify-golint.sh check return
```
pkg\quota\v1\evaluator\core\doc.go:17:1: package comment should be of the form "Package core ..."
```

File: pkg/registry/admissionregistration/mutatingwebhookconfiguration/storage/storage.go [hunk 2]
```diff
@@ -24,7 +24,7 @@ import (
 	"k8s.io/kubernetes/pkg/registry/admissionregistration/mutatingwebhookconfiguration"
 )
 
-// rest implements a RESTStorage for pod disruption budgets against etcd
+// REST implements a RESTStorage for pod disruption budgets against etcd
```
Comment (xichengliudui) [inline pkg/registry/admissionregistration/mutatingwebhookconfiguration/storage/storage.go]: why change: using verify-golint.sh check return
```
pkg\registry\admissionregistration\validatingwebhookconfiguration\storage\storage.go:27:1: comment on exported type REST should be of the form "REST ..."
```

**Comment by xichengliudui** [comment]
@derekwaynecarr @BenTheElder , do we still need to merge this pr? I'll close it if I don't need to merge it.

**Comment by spiffxp** [comment]
/priority important-longterm
/approve
for the hack/ changes, I don't think I have /approve rights in the other files

**Pull request closed without merging**